CHANGELOG.md

```
@@ -1,98 +1,96 @@
 
-#### 10.2.4 - November 2024
+#### 9.5.4 - November 2024
   * Rolling back SHA-256 checksum algorithm to MD5 in the MT message model
 
-#### 10.2.3 - November 2024
+#### 9.5.3 - November 2024
   * (PW-2040) Updated the BBAN validation data file to the IBAN REGISTRY Jul 2024 release 
   * (PW-2006) Fixed `getMUR` and `setMUR` in `SwiftMessage` to prioritize field 108 in block 4 over block 3 for system messages (category 0)
   * Added new `MtSequenceEnum` with all the available inner sequences of specific MT schemas
   * Added `isSystemMessage()` to SwiftMessage to check if the message is a category 0 message (010. 011, etc...)
+  * Added new `MtSequenceEnum` with all the available inner sequences of specific MT schemas
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)